index.html

```diff
@@ -152,6 +152,33 @@ <h3>Skills</h3>
             </ul>
           </div>
         </section>  
+        <!--Get Started Section-->
+      <section id="get-started">
+        <img class="get-started-img" src="images/image_geometry_1.svg" alt="image_geometry_3">
+        <div class="container">
+          <h2>Get started</h2>
+          <p class="get-started-content">
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna.
+        </p>
+        <form action="https://formspree.io/f/mlezegyq" method="POST">
+            <input type="text" id="name" name="user_name" maxlength="30" placeholder="Your name" required/>
+            <input type="email" id="email" name="user_email" placeholder="Your email" required/>
```

Comment:
- [ ] Please kindly try to match the Figma as closely as you can. notice that the "Start collaboration" button on the Figma design doesn't have a border. On your design you still have a border visible. please kindly remove it. 
## Figma Design
![image](https://user-images.githubusercontent.com/18730106/149023041-de2ab809-e556-420b-93e9-037dbf3531e4.png)

## Your Design 
![image](https://user-images.githubusercontent.com/18730106/149023092-adc140c7-b600-44a6-8147-f922bfc2bcc4.png)

Reply:
Hi @MrBrN197, I don't know what's happened with the button, but if you pay attention again to my CSS file you will see no border except when it 's pressed.